website/payments/services.py

```diff
@@ -158,7 +158,7 @@ def process_batch(batch):
             bank_account.last_used = batch.withdrawal_date
             bank_account.save()
 
-    batch.save()
+    batch.save(update_fields=["processed"])
 
```

Comment:
Why do we change this? I think this also breaks some logic in the batch's `save()` method that tries to set the `processed_at` field. 

If you want to know which field was updated in the signal, please detect it yourself.

Reply:
I tried but `update_fields` doesn't show a value if this is not set. I also couldn't find a different way to find out if the batch went to `processed` during the save

Reply:
I just realized the other approach only works with pre_save signals. Maybe create a `PushedThaliaPayBatch` model in the moneybirdsynchronization app and check at each signal if it was pushed already